### ISSUE TITLE
Improve our authorship policy?

### ISSUE THREAD

**Issue by mrshirts** [livecomsjournal/livecomsjournal.github.io#137]
See https://blog.scholasticahq.com/post/implementing-authorship-policy/ for suggestions about an authorship policy.

**Comment by davidlmobley** [comment]
I agree this is something we want to address (cc @dmzuckerman ). As per an e-mail exchange, our approach currently is basically that "authors must agree amongst themselves", which I largely agree with, so our policy needn't be complex. However, there are some issues raised by the article above which are worth addressing. For example, it notes several types of problematic authorship:

> Guest authorship: an honorary or courtesy authorship granted out of respect to a researcher or in order to use his or her standing to boost a paper’s credibility
> Gift authorship: authorship credit granted out of a sense of obligation to a researcher or to benefit him or her
> Ghost authorship: neglecting to identify a rightful author in a paper

I think we want to avoid all of those. Here, we want authors to be those people (and ALL those people) who contributed substantially to the papers we are publishing, and not anyone else. 

Imagine a student or a postdoc writes part of a contribution for LiveCoMS with the approval of their supervisor, but without any involvement of the supervisor. In a "normal" journal I think this MIGHT be grounds for including the PI as an author, simply because the PI might have provided financial support. However, I think here we will usually not want the PI as an author unless the PI has actually been involved in the writing (or planning of research, if research is reported), so we probably want that to be clear. We don't want to be in a position (since we may already have lots of authors) where we pick up lots of bonus authors just because of who is supervising whom, or who used to supervise them, or...

**Comment by davidlmobley** [comment]
@mrshirts @dmzuckerman - any more thoughts on authorship policy? Should I draft something on this?

**Comment by mrshirts** [comment]
We could use the CRediT taxonomy: http://docs.casrai.org/CRediT

**Comment by dmzuckerman** [comment]
CRediT very explicit, which is great, but note that getting funding qualifies for authorship under the CrediT scheme, which is directly contrary to the wish of @davidlmobley

**Comment by davidlmobley** [comment]
I guess for some categories of articles "getting funding" could be sufficient, eg for lessons learned. But mostly we want authors who are going to be engaged with the article.

Since these are mostly not research articles, I think perhaps the CRediT taxonomy may not be the best option.

**Comment by dmzuckerman** [comment]
Well, we can just make our own version.  Below I cut and paste their list of contribution types, with some we may want to omit/make conditional ... 

Contributor Roles/Conceptualization
Contributor Roles/Data curation - ONLY IF INTENSIVE
Contributor Roles/Formal analysis
Contributor Roles/Funding acquisition - DAVID SAYS NO; I AGREE
Contributor Roles/Investigation
Contributor Roles/Methodology - ONLY IF METHODOLOGY DEVELOPED FOR REPORTED DATA AND NOT YET PUBLISHED - NO DOUBLE-DIPPING!
Contributor Roles/Project administration - ONLY IF INTENSIVE
Contributor Roles/Resources - I DON'T LIKE THIS ONE
Contributor Roles/Software
Contributor Roles/Supervision
Contributor Roles/Validation
Contributor Roles/Visualization
Contributor Roles/Writing – original draft
Contributor Roles/Writing – review & editing

**Comment by davidlmobley** [comment]
@dmzuckerman - I wonder if "Supervision" should be labeled "only if the article reports a research project or is otherwise exceptional in some way". "I supervised my students writing a review paper" is not a significant contribution in my opinion, unless you read it/edited it. :) 

And I otherwise agree with your comments.

**Comment by mrshirts** [comment]
So, we could say we are using CRediT, with the modification we list below . . .

**Comment by davidlmobley** [comment]
I agree with Dan, @mrshirts .
